app/models/subscription.rb

```diff
@@ -5,6 +5,8 @@ class Subscription < ApplicationRecord
   belongs_to :course
   has_many :lessons, through: :course
 
+  validates :user, uniqueness: true
```

Comment:
Doesn't this mean that each user can just have a single Subscription? I think only the combination of user and course should be unique.

Reply:
😱 you're totally right, thanks 😄